build/dockerfiles/Dockerfile

```diff
@@ -17,13 +17,8 @@ RUN ./check_mandatory_fields.sh devfiles
 
 RUN ./index.sh > /build/devfiles/index.json
 RUN ./list_referenced_images.sh devfiles > /build/devfiles/external_images.txt
-RUN chmod -R g+rwX /build/devfiles
-
-FROM docker.io/node:16.13.0-alpine3.14 as dwtemplates
-COPY ./build/scripts /build/
```

Comment:
these three lines are dupes of work being done higher up, so if we start with `node:16.13.0-alpine3.14` instead of outdated `alpine:3.11.5`, we don't need a second stage here.